src/__tests__/__snapshots__/transform-test.js.snap

```diff
@@ -28,14 +28,25 @@ exports[`.transform(type definitions) 1`] = `
         validateXDR(input: string, format: \\"hex\\" | \\"base64\\"): boolean;
     }
 
+    interface Bool {
+        read(io: Buffer): boolean;
+        write(value: boolean, io: Buffer): void;
+        isValid(value: boolean): boolean;
+        toXDR(value: boolean): Buffer;
+        fromXDR(input: Buffer, format?: \\"raw\\"): boolean;
+        fromXDR(input: string, format: \\"hex\\" | \\"base64\\"): boolean;
+        validateXDR(input: Buffer, format?: \\"raw\\"): boolean;
+        validateXDR(input: string, format: \\"hex\\" | \\"base64\\"): boolean;
+    }
```

Comment:
It might be nice to implement `valueOf()` for these types. (Hyper as well could do with a `valueOf(): bigint`). It would make using them slightly more ergonomic because you could do stuff like:
```ts
Bool.fromXDR("...") == true
```
or
```ts
new Hyper("123") + 5n // 128n
```
But that's probably a separate PR. Mind opening an issue to track that? Not sure if that should be here, or in base, or where.

Reply:
Good call! I didn't know `valueOf` could be leveraged that way, TIL! I made stellar/js-xdr#101, which I think is the best place for it.